Readme.md

```diff
@@ -137,26 +137,25 @@ to send an email. The `userId` represents the authenticated user and can be a Gm
 
 ```ballerina
 string userId = "me";
-
-gmail:MessageRequest messageRequest = {};
-messageRequest.recipient = "recipient@mail.com";
-messageRequest.sender = "sender@mail.com";
-messageRequest.cc = "cc@mail.com";
-messageRequest.subject = "Email-Subject";
-messageRequest.messageBody = "Email Message Body Text";
-
-//Set the content type of the mail as TEXT_PLAIN or TEXT_HTML.
-messageRequest.contentType = gmail:TEXT_PLAIN;
+gmail:MessageRequest messageRequest = {
+    recipient : "aa@gmail.com",
+    sender : "bb@gmail.com",
+    cc : "cc@gmail.com",
+    subject : "Email-Subject",
+    messageBody : "Email Message Body Text",
+    // Set the content type of the mail as TEXT_PLAIN or TEXT_HTML.
+    contentType : gmail:TEXT_PLAIN
+};
 ```
 
 ### Step 4: Send the message
-The response from `sendMessage` is either a MessageResponse record or an `error` (if sending the message was unsuccessful).
+The response from `sendMessage` is either a Message record or an `error` (if sending the message was unsuccessful).
 
 ```ballerina
 //Send the message.
-gmail:MessageResponse|error sendMessageResponse = checkpanic gmailClient->sendMessage(userId,messageRequest);
+gmail:Message|error sendMessageResponse = checkpanic gmailClient->sendMessage(userId,messageRequest);
```

Comment:
What is the reason for `checkpanic`? I could see it from down also. Please reply with the Git commit or PR If this is already has been fixed.

Reply:
This code snippet is moved to Module.md. The correction is in following commit. : https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail/commit/5a1d71c7f012e587bed57a1e5d58905d006ef3fa